core/repos/repomanager.go

```diff
@@ -454,7 +463,7 @@ func (manager *RepoManager) GetWorkflow(workflowPath string)  (resolvedWorkflowP
 	if !strings.HasSuffix(workflowFile, ".yaml") { //Add trailing ".yaml"
 		workflowFile += ".yaml"
 	}
-	resolvedWorkflowPath = workflowRepo.getWorkflowDir() + workflowFile
+	resolvedWorkflowPath = filepath.Join(workflowRepo.getWorkflowDir(), workflowFile)
```

Comment:
Thanks to the fact that you use `filepath.Join`, this will work regardless of whether `getWorkflowDir` returns a trailing slash or not. But other context of `getWorkflowDir` should be checked too.